CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 #### Visual Environment
 
+- [Signed builds.][1366] Our builds are signed and will hav fewer annoyances
+  upon running. You can trust us.
```

Comment:
1. It contains errors.
2. "You can trust us." sounds rather not professional. I'd rewrite it.